mozdownload/scraper.py

```diff
@@ -870,13 +870,130 @@ def platform_regex(self):
         return PLATFORM_FRAGMENTS[self.platform]
 
 
+class TryScraper(Scraper):
+    "Class to download a try build from the Mozilla server."
+
+    def __init__(self, changeset=None,
+                 debug_build=False, *args, **kwargs):
+
+        self.debug_build = debug_build
+        self.changeset = changeset
+
+        Scraper.__init__(self, *args, **kwargs)
+
+    def get_build_info(self):
+        "Defines additional build information"
+
+        self.build_index = None
+
+        self.locale_build = self.locale != 'en-US'
```

Comment:
I followed the pattern that [TinderboxScraper uses](https://github.com/mozilla/mozdownload/blob/master/mozdownload/scraper.py#L744), in retrospect I don't think it is necessary for try builds, I'll remove it.